src/easylogging++.cc

```diff
@@ -1128,7 +1128,7 @@ unsigned long long DateTime::getTimeDifference(const struct timeval& endTime, co
     return static_cast<unsigned long long>(static_cast<unsigned long long>(1000000 * endTime.tv_sec + endTime.tv_usec) -
                                            static_cast<unsigned long long>(1000000 * startTime.tv_sec + startTime.tv_usec));
   } else {
-    return static_cast<unsigned long long>((((endTime.tv_sec - startTime.tv_sec) * 1000000) +
+    return static_cast<unsigned long long>((((endTime.tv_sec - startTime.tv_sec) * 1000) +
```

Comment:
To calculate in microsecond precision, it is good to have seconds * 1000000 + microseconds.

To calculate in millisecond precision, obviously it should be seconds * 1000 + milliseconds. Currently you had "seconds * 1000000 + milliseconds", which basically makes the milliseconds insignificant and result wrong. One second is equal to 1000 milliseconds, not 1000000 milliseconds.